Automattic/wp-calypso
# Site Importer: Show siteUrl in the preview step

#### Changes proposed in this Pull Request

* Include the source site's URL in the "Is this your site?" / "Are you sure" screen.
* styling

#### Testing instructions

1. Visit the importers section of a site (`/settings/import/{site-slug}`)
1. Choose the Wix importer
1. Enter a valid url to import and click "Continue" (example: https://hi6822.wixsite.com/eat-here-its-good)
1. You should see the URL to import above the preview image
1. Clicking on the URL opens the link in a new tab/window

<img width="665" alt="screen shot 2018-10-29 at 15 52 08" src="https://user-images.githubusercontent.com/1699996/47679615-a6814d00-db92-11e8-887f-c98fb1152e90.png">
